src/python/gudhi/point_cloud/timedelay.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+class TimeDelayEmbedding:
+    """Point cloud transformation class.
+
+    Embeds time-series data in the R^d according to Takens' Embedding Theorem
+    and obtains the coordinates of each point.
```

Comment:
It may not be completely obvious for a user what `delay` and `skip` really mean, I believe that a simple example (where delay and skip are different and not 1) here (or in point_cloud.rst), similar to the one in the tests, would help users understand immediately what it does.

Reply:
I understood to write such an example.  
is it right?  
```
[For example]
time_series = [1, 2, 3, 4, 5, 6, 7]  
Given skip=2, delay=2, the attractor will be like this.
attractor = [[1, 3, 5], 
             [3, 5, 7]]
```

Reply:
Yes, you can add those few lines to the docstring, just like that, but choosing values skip != delay (maybe 2 and 3) so that users don't confuse them.
If you want to add it in the .rst file instead, it can be presented as code (see for instance wasserstein_distance_user.rst which uses testcode and testoutput), where the testsuite actually executes the 2 lines of code and checks that the output hasn't changed.
I let you choose if you prefer to put it in timedelay.py or point_cloud.rst.